bin/run.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+usage() {
+  echo "usage: ./bin/run.sh uwsgistart|start|bash|whatevercommandyouwant"
+  exit 1
+}
+
+[ $# -lt 1 ] && usage
+
+case $1 in
+  migrate)
+    kinto migrate --ini config/example.ini
+    ;;
+  start)
+    pushd mail
+    rm -fr *.eml
+    python mailfileserver.py --bind 0.0.0.0 9999 &
+    popd
+
+    kinto start --ini config/example.ini
+    ;;
+  uwsgistart)
+    pushd mail
+    rm -fr *.eml
+    python mailfileserver.py --bind 0.0.0.0 9999 &
+    popd
+
+    KINTO_INI=config/example.ini uwsgi --http :8888 --ini config/example.ini
```

Comment:
in this file I believe we could use a global env variable `KINTO_INI` instead of repeating `config/example.ini`

Reply:
Done.